browser/components/MarkdownEditor.js

```diff
@@ -275,7 +275,8 @@ class MarkdownEditor extends React.Component {
           noteKey={noteKey}
           fetchUrlTitle={config.editor.fetchUrlTitle}
           enableTableEditor={config.editor.enableTableEditor}
-          onChange={(e) => this.handleChange(e)}
+          linesHighlighted={linesHighlighted}
+          onChange={(e) => this.handleChange.bind(this)(e)}
```

Comment:
You don't need to `bind(this)` in an arrow function, the `this` keyword will point to the class if you use arrow function.

Reply:
Didn't know that, thank you for the correction!